cache/deno.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "@std/cache",
+  "version": "0.224.1",
```

I think this should be 0.1.0.
```suggestion
  "version": "0.1.0",
```